scripts/linters/pylint_extensions.py

```diff
@@ -2919,6 +2919,48 @@ def visit_importfrom(self, node: astroid.nodes.ImportFrom) -> None:
                 self.add_message('disallowed-text-import', node=node)
 
 
+# TODO(#16567): Here we use MyPy ignore because of the incomplete typing of
+# pylint library and absences of stubs in pylint, forces MyPy to
+# assume that BaseChecker class has attributes of type Any.
+# Thus to avoid MyPy's error
+# (Class cannot subclass 'BaseChecker' (has type 'Any')),
+# we added an ignore here.
+class BlankLineBelowFunctionDefChecker(checkers.BaseChecker):  # type: ignore[misc]
+    """Checks if there a single blank line after function definition."""
+
+    __implements__ = interfaces.IAstroidChecker
+
+    name = 'space_after_function_def'
+    priority = -1
+    msgs = {
+        'C0041': (
+            'Blank line after function definition',
+            'Blank-line-after-function-definition',
+            'Please remove blank line after function definition',
+        ),
+    }
+
+    def visit_module(self, node: astroid.scoped_nodes.Module) -> None:
+        """Visit a module to ensure that there is a blank line below
+        file overview docstring.
```

Comment:
Is this right? I think it should say that we're checking that there is no blank line after a function definition, right?